Firebase/android/app/src/main/res/layout/merge_message_item_view.xml

```diff
@@ -2,55 +2,44 @@
 <merge xmlns:android="http://schemas.android.com/apk/res/android">
 
   <LinearLayout
+    style="@style/Chat.Message.Metadata"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:orientation="horizontal"
-    android:paddingLeft="70dp">
+    android:layout_height="wrap_content">
 
     <TextView
       android:id="@+id/messageAuthorName"
+      style="@style/Chat.Message.Author"
       android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:fontFamily="sans-serif-medium"
-      android:textColor="@color/text_grey"
-      android:textSize="10sp" />
+      android:layout_height="wrap_content" />
 
     <TextView
       android:id="@+id/messageTime"
+      style="@style/Chat.Message.Time"
       android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:paddingLeft="8dp"
-      android:textColor="@color/text_grey_light"
-      android:textSize="10sp" />
+      android:layout_height="wrap_content" />
 
   </LinearLayout>
 
   <RelativeLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:paddingTop="6dp">
+    android:paddingTop="@dimen/chat_message_body_padding">
 
     <ImageView
       android:id="@+id/messageAuthorImage"
```

Comment:
are mixed case allowed for IDs? :S underscore naming is the style I have seen and used in all the projects I've worked on so far at Novoda.